examples/escrow/Cargo.toml

```
@@ -0,0 +1,21 @@
+[project]
+authors = ["Braqzen"]
```

For both Cargo and Forc manifest files.
```suggestion
authors = ["Fuel Labs <contact@fuel.sh>"]
```